DESCRIPTION

```diff
@@ -41,6 +41,7 @@ Suggests:
     knitr,
     mapview,
     readr,
+    rerddap,
```

Comment:
why list it here? maybe rather in a special field à la Config/Needs see https://r-pkgs.org/description.html#sec-description-custom-fields?